sources/platform/storage/request_queue.md

```diff
@@ -274,6 +274,195 @@ Example payload:
 
 For further details and a breakdown of each storage API endpoint, refer to the [API documentation](/api/v2#/reference/key-value-stores).
 
+## Features {#features}
+
+Request queue is a storage type built with scraping in mind, enabling developers to write scraping logic efficiently and scalable.
+The Apify tooling, including [Crawlee](https://crawlee.dev/) and [Apify SDKs](https://docs.apify.com/sdk/js/), incorporates all these features, enabling users to leverage them effortlessly without extra configuration.
+
+In the following section, we will discuss each main features in depth.
+
+### Persistence and retention {#persistence-and-retention}
```

Comment:
```suggestion
### Persistence and retention
```

If there is no difference between heading and anchor in curly brackets we can omit it. and Docusaurus will generate anchor based on the heading